backend/src/components/facilities.js

```diff
@@ -1,14 +1,56 @@
 'use strict'
 const { getOperation, patchOperationWithObjectId } = require('./utils')
 const { MappableObjectForFront, MappableObjectForBack } = require('../util/mapping/MappableObject')
-const { FacilityMappings, RoleMappings, UserMappings, UsersPermissionsFacilityMappings, LicenceMappings } = require('../util/mapping/Mappings')
+const { FacilityMappings, RoleMappings, UserMappings, UsersPermissionsFacilityMappings, LicenceMappings, FacilityAdditionalAddressMappings } = require('../util/mapping/Mappings')
 const HttpStatus = require('http-status-codes')
 
+/**
+ * Dynamics can take a selector with a list of fields
+ * Use this function to generate a list of fields based on mappings
+ */
+function getMappingString(mappings) {
+  let retVal = mappings
```

Comment:
You can write this as a pretty clear single line if you skip the variable declaration and use an implicit return.
This might be useful elsewhere in the code...can we put it somewhere common?

Reply:
done! Yes I thought you might want to use it in other places. Could be worth refactoring some of the giant strings to use this instead at some point

Reply:
You can simplify this further with an implicit return:

`return mappings.map((item) => item.back).join(',')`

Reply:
yup done now